util/src/main/java/io/kubernetes/client/util/Config.java

```diff
@@ -104,7 +104,15 @@ public static ApiClient fromConfig(Reader input) {
 
     public static ApiClient fromConfig(KubeConfig config) {
         ApiClient client = new ApiClient();
-        client.setBasePath(config.getServer());
+        String server = config.getServer();
+        if (!server.startsWith("http://") && !server.startsWith("https://")) {
+            if (server.indexOf(":443") != -1) {
```

Comment:
Should we rely on `config.verifySSL()` instead/as well?

Reply:
verifySSL is default true, so I don't think that's the right thing to test here...

Reply:
Ah good point.